### PR TITLE
[qa] Move CI build to GitHub Actions

### LINKED ISSUE #199: [openwisp-radius] CI: move to Github Actions
Let's move to Github Actions here too.

### PR DESCRIPTION
**Pull request by pandafy**
Removed Travis

Fixes #199

### CODE REVIEW

**Review by nemesisdesign — COMMENTED** [review]
@pandafy do you know why the build is failing at the 404 check part?

**Review by nemesisdesign — COMMENTED** [review]
Check this out as well: https://github.com/openwisp/openwisp-radius/blob/master/run-qa-checks#L9-L17

**Comment by pandafy** [comment]
> Check this out as well: https://github.com/openwisp/openwisp-radius/blob/master/run-qa-checks#L9-L17

Thanks for the heads up @nemesisdesign. I did not expect something like that in `run-qa-checks`. I have updated that code so it is not only bound to any CI. 

I did not get much time before to debug this. Since Travis started working again, this PR lost my priority. :sweat_smile:

File: run-qa-checks [hunk 1]
```diff
@@ -6,7 +6,8 @@ echo ''
 echo 'Checking documentation build status'
 ./runsphinx-build
 
-if [[ $TRAVIS_PYTHON_VERSION != 3.6 ]]; then
+PYTHON_VERSION=$(python -c 'import sys; print(".".join(map(str, sys.version_info[:2])))')
+if [[ $PYTHON_VERSION != 3.6 ]]; then
```
Comment (pandafy) [inline run-qa-checks:10]: This code makes it independent of the platform it runs on. Sure, it does not look pretty but, it gets the job done.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        run: |
+          pip install -U pip wheel
+          pip install ${{ matrix.django-version }}
+          pip install -U -r requirements-test.txt
+
+      - name: Install npm dependencies
+        run: sudo npm install -g jslint
+
+      - name: Install openwisp-radius
+        run: pip install -e .
```
Comment (pandafy) [inline .github/workflows/ci.yml:48]: Swapped out `python setup.py -q develop` as it was not resolving `djangorestframework` dependency properly. Somehow, `djangorestframework==3.12.2` was getting installed in the environment.

File: run-qa-checks (hunk 1 above)
Comment (nemesisdesign) [reply run-qa-checks:10]: Is great, thanks

**Review by nemesisdesign — APPROVED** [review]
Great work @pandafy :+1:

**Pull request merged**